src/transaction/perform-transaction-composer-simulate.ts

```diff
@@ -0,0 +1,24 @@
+import { RawSimulateOptions, SimulateOptions, TransactionComposer } from '../types/composer'
+
+/**
+ * @deprecated Use `composer.simulate` with `allowEmptySignatures` flag set to true
+ *
+ * Performs a simulation of the transactions loaded into the given TransactionComposer.
+ * Uses empty transaction signers for all transactions.
+ *
+ * @param composer The TransactionComposer with transaction(s) loaded.
+ * @returns The simulation result, which includes various details about how the transactions would be processed.
+ */
+export async function performTransactionComposerSimulate(composer: TransactionComposer, options?: RawSimulateOptions) {
+  // NOTE: the existing version takes atc as params, then call atc.buildGroup to get the transactions
+  // The state of the atc is unknown, whether it has resource populated or not
+  // In this version, we call composer.simulate which doesn't do resource population
```

Comment:
I would have thought that `build()` would need to be called and as a result it would have been ran? I don't think we want to run resource pop on simulate though, as it essentially results in 2 simulates.